### ISSUE TITLE
License

### ISSUE THREAD

**Issue by MagicalTux** [otiai10/primes#9]
This repository doesn't have a LICENSE file, resulting in godoc refusing to show:

https://pkg.go.dev/github.com/otiai10/primes?utm_source=godoc

    Documentation not displayed due to license restrictions.

@otiai10 Adding a license file would solve the issue.

**Comment by otiai10** [comment]
Thank you. I'll do that.
Btw, how did you find this small package? Just curious.

**Comment by MagicalTux** [comment]
@otiai10 This package is used by a crypto exchange, Binance, in a critical library that handles private keys and signatures.

Specifically, it is used to generate a list of the 1000 first primes that is then used in a [Paillier cryptosystem](https://en.wikipedia.org/wiki/Paillier_cryptosystem).

Source: https://github.com/bnb-chain/tss-lib/blob/master/crypto/paillier/paillier.go#L28

I have forked this library for various reasons, including removing as many dependencies as I can, and [I have replaced the use of your library by including a list of the prime numbers below 1000](https://github.com/ModChain/tss-lib/commit/68d147b012dbda1610ef4231544db9094daa89ac).

Since that's all it's used for I don't think there's really a need to keep your lib in, but I'd still like to thank you for your service.
